packages/terra-functional-testing/src/commands/validates/screenshot.js

```diff
@@ -19,8 +19,8 @@ const screenshot = (testName, options = {}) => {
     ...options,
     name: testName,
   };
-
   const screenshotResult = global.browser.checkElement(selector || global.Terra.serviceOptions.selector, wrappedOptions);
```

Comment:
personal preference, but we could be more declarative by returning a new object instead of appending a new property

```suggestion
  const checkElementResult = global.browser.checkElement(selector  || global.Terra.serviceOptions.selector);
  const disableSnapshotMismatch = global.Terra.serviceOptions.disableSnapshotMismatch;
  const result = {
    ...checkElementResult,
    disableSnapshotMismatch,
  };
  
  global.expect(result).toMatchReference();
```
```

Reply:
We should just be able to check for `global.Terra.serviceOptions.disableSnapshotMismatch` directly in [toMatchReference](https://github.com/cerner/terra-toolkit/pull/655/files#diff-1372ed3653dff72eae9a8aa8b3969b4e50473f645766c7c2f232298e514d0518R40) instead of adding it to the result and then pull it out later.

Reply:
Changed here https://github.com/cerner/terra-toolkit/pull/655/commits/8571fc4e7f0de6b5aa3178eb5c70084c3fa80539#diff-1372ed3653dff72eae9a8aa8b3969b4e50473f645766c7c2f232298e514d0518R39